File: app/src/main/java/com/crux/bphcfreshers/CInfoFragment.java
package com.crux.bphcfreshers;


import android.content.Intent;
import android.os.Bundle;
import androidx.annotation.NonNull;
import androidx.annotation.Nullable;
import androidx.fragment.app.Fragment;
import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;
import android.widget.Button;

public class CInfoFragment extends Fragment implements View.OnClickListener {

    @Nullable
    @Override
    public View onCreateView(@NonNull LayoutInflater inflater, @Nullable ViewGroup container, @Nullable Bundle savedInstanceState) {

        return inflater.inflate(R.layout.fragment_cinfo, null);

//        return super.onCreateView(inflater, container, savedInstanceState);
    }

    @Override
    public void onViewCreated(@NonNull View view, @Nullable Bundle savedInstanceState) {
        super.onViewCreated(view, savedInstanceState);

        Button Bhistory = view.findViewById(R.id.button5);
        Button Bc_d = view.findViewById(R.id.button6);
        Button Bnearby = view.findViewById(R.id.button7);
        Bhistory.setOnClickListener(this);
        Bc_d.setOnClickListener(this);
        Bnearby.setOnClickListener(this);


    }

    @Override
    public void onClick(View v) {
        switch (v.getId()) {
            case R.id.button5:
                openHistory();
                break;
            case R.id.button6:
               openClub();
                break;
            case R.id.button7:
               openFests();
                break;
        }
    }

    public void openHistory() {
        Intent intent = new Intent(CInfoFragment.this.getActivity(), History.class);
        startActivity(intent);


    }
    public void openClub() {
        Intent intent = new Intent(CInfoFragment.this.getActivity(), clubdep.class);
        startActivity(intent);


    }

    public void openFests() {
        Intent intent = new Intent(CInfoFragment.this.getActivity(), Fests.class);
        startActivity(intent);


    }


}

File: app/src/main/java/com/crux/bphcfreshers/CInfoFragment.kt
package com.crux.bphcfreshers

import android.content.Intent
import android.os.Bundle
import android.view.LayoutInflater
import android.view.View
import android.view.ViewGroup
import android.view.animation.AnimationUtils
import android.widget.Button
import android.widget.ImageView
import android.widget.TextView
import androidx.fragment.app.Fragment

class CInfoFragment : Fragment(), View.OnClickListener {
    override fun onCreateView(inflater: LayoutInflater, container: ViewGroup?, savedInstanceState: Bundle?): View? {
        return inflater.inflate(R.layout.fragment_cinfo, null)
        //        return super.onCreateView(inflater, container, savedInstanceState);
    }

    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
        super.onViewCreated(view, savedInstanceState)
        val Bhistory = view.findViewById<Button>(R.id.button5)
        val Bc_d = view.findViewById<Button>(R.id.button6)
        val Bnearby = view.findViewById<Button>(R.id.button7)
        val Blogo=view.findViewById<ImageView>(R.id.blogo);
        val cinftxt=view.findViewById<TextView>(R.id.textView3);
        val logoanim= AnimationUtils.loadAnimation(context,R.anim.logoanim);
        val bttnanim=AnimationUtils.loadAnimation(context,R.anim.bttnanim);
        Blogo.startAnimation(logoanim);
        cinftxt.startAnimation(logoanim);
        Bhistory.startAnimation(bttnanim);
        Bc_d.startAnimation(bttnanim);
        Bnearby.startAnimation(bttnanim);

        Bhistory.setOnClickListener(this)
        Bc_d.setOnClickListener(this)
        Bnearby.setOnClickListener(this)
        Bhistory.setOnClickListener(this)
        Bc_d.setOnClickListener(this)
        Bnearby.setOnClickListener(this)
    }

    override fun onClick(v: View) {
        when (v.id) {
            R.id.button5 -> openHistory()
            R.id.button6 -> openClub()
            R.id.button7 -> openFests()
        }
    }

    fun openHistory() {
        val intent = Intent(this@CInfoFragment.activity, History::class.java)
        startActivity(intent)
    }

    fun openClub() {
        val intent = Intent(this@CInfoFragment.activity, clubdep::class.java)
        startActivity(intent)
    }

    fun openFests() {
        val intent = Intent(this@CInfoFragment.activity, Fests::class.java)
        startActivity(intent)
    }
}